app/views/Dashboard/CombinedIndicators/TopicCard/index.tsx

```diff
@@ -20,12 +21,15 @@ const subvariableKeySelector = (d: { emergency: string }) => d.emergency;
 
 interface Props {
     topicName: string;
-    topicDescription?: string;
+    topicDescription: string | undefined;
     indicators: IndicatorType[] | undefined;
     showRegionalValue: boolean;
     filterValues: FilterType | undefined;
     setFilterValues: React.Dispatch<React.SetStateAction<FilterType | undefined>>;
     setActiveTab: React.Dispatch<React.SetStateAction<TabTypes | undefined>>;
+    includedIndicators: (string | undefined)[] | undefined;
+    includedSubvariables: (string | undefined)[] | undefined;
```

Comment:
```suggestion
    includedIndicators: string[] | undefined;
    includedSubvariables: string[] | undefined;
```

Filter the undefined ones before sending them here.